content/en/tracing/setup/java.md

```diff
@@ -112,6 +113,7 @@ Don't see your desired web frameworks? Datadog is continually adding additional
 | AWS Java SDK             | 1.11+, 2.2+ | Fully Supported | `aws-sdk`                                      |
 | Commons HTTP Client      | 2.0+        | Fully Supported | `commons-http-client`                          |
 | Google HTTP Client       | 1.19.0+     | Fully Supported | `google-http-client`                           |
+| Grizzly HTTP Client      | 1.9+        | Beta[8]         | `grizzly-client`                               |
```

Comment:
I think we have `ning` as an option too.

Reply:
Done